### PR TITLE
variables updated to v2

### PR DESCRIPTION
**Pull request by kirsty-hames**

### CODE REVIEW

File: less/accordion.less [hunk 1]
```diff
@@ -1,87 +1,87 @@
 .accordion-component {
 
     .accordion-item {
-        margin-bottom:@item-spacing;
+        margin: @item-margin-bottom;
```
Comment (lc-thomasberger) [inline less/accordion.less]: should this be margin-bottom?

**Comment by guywillis** [comment]
+1

**Comment by hrajotia** [comment]
+1

**Comment by moloko** [comment]
+1

**Pull request merged**